packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -180,92 +181,29 @@ impl<'content> AssetRouter<'content> {
     /// given [HttpRequest](ic_http_certification::HttpRequest).
     pub fn serve_asset(
         &self,
+        data_certificate: &[u8],
```

Comment:
Please document the added argument, maybe with a reference to the spec about the format, and mentioning `ic_cdk::api::data_certificate()` as a usual source for the value (as you've mentioned in the code example above).

Reply:
Good point, thanks.